values.yaml

```diff
@@ -425,94 +425,129 @@ st2chatops:
 ## MongoDB HA configuration (3rd party chart dependency)
 ##
 ## For values.yaml reference:
-## https://github.com/helm/charts/tree/master/stable/mongodb-replicaset
+## https://github.com/bitnami/charts/tree/master/bitnami/etcd
```

Comment:
Looks like we refer to `etcd` helm chart here instead of `mongodb` by mistake.

Reply:
I've fixed this one up